sdk/resourcemanager/azure-resourcemanager/pom.xml

```diff
@@ -57,6 +57,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
what's the change?

this likely breaks downstream

Reply:
1. changed `CategoryType` from enum to class
2. removed some classes
3. removed `JsonFlatten` annotation